cogs/clash_river_war.py

```diff
@@ -245,15 +279,20 @@ async def clanRemainingWarAttacks(self, ctx, clan_tag: str):
             all_clan_attacks = self.clash_royale_service.clan_remaining_war_attacks(clan_tag)
         embed = remaining_war_embed(all_clan_attacks)
         await ctx.send(embed=embed)
-    @riverwar.error
-    async def riverwar_error(self, ctx, error):
-        if isinstance(error, commands.MissingRequiredArgument):
-            await ctx.send(embed=discord.Embed(
-                description="Incorrect arguments entered.\nCorrect usage: **!riverwar [clan_tag] [past_weeks]**" +
-                "\nOr use the AUSCLAN version: **!ausclanwar**",
-                colour=discord.Colour.red()
-            ))
 
+    @commands.command(name="ausclanwarplayers", pass_context=True)

```

Comment:
Let's add an argument to this command to filter out players who left clan. Seems like a useful feature to have if ppl want to just remind those still in clan to attack. Wdyt?

Reply:
I've got a flag already that checks whether the person left the clan or not! Adding filter means more typing and it might make it too complicated for them

Reply:
Done.